README.md

```diff
@@ -510,18 +479,18 @@ https://mmancusa.webex.com/mmancusa/ldr.php?RCID=e5e6fc5b6d218307f9eec863111e603
 - Improved text output (\#12)
 - UGRID compliance (\#33)
 
-## Contributors    
+## Contributors
 
 - Dave Foster <dave@axiomdatascience.com>
 - Dan Maher <dmaher@asascience.com>
 - Luke Campbell <lcampbell@asascience.com>
 - [Kyle Wilcox](https://github.com/kwilcox) <kyle@axiomdatascience.com>
 - Ben Adams <ben.adams@rpsgroup.com>
 
-And many more testers!    
+And many more testers!
+
+Portions of the CF checker are based on Michael Decker's work, http://repositories.iek.fz-juelich.de/hg/CFchecker/
 
-Portions of the CF checker are based on Michael Decker's work, http://repositories.iek.fz-juelich.de/hg/CFchecker/    
- 
 <!--
```

Comment:
er, this doesn't look right in the summary view, i'm talking about the commented out block of commands.